package.json

```diff
@@ -15,7 +15,7 @@
     "jest-expo": "^27.0.1",
     "jsdoc": "^3.5.5",
     "prop-types": "^15.6.1",
-    "react-test-renderer": "^16.3.2"
+    "react-test-renderer": "16.3.0"
```

Comment:
Hi @gkemp94 , 

this is done to avoid the upgrade to the buggy version, right?

Reply:
Yep Exactly, there were a couple things that weren't compatible.

Reply:
Any chance you could go into more detail (list) on what made that version to be incompatible with our app?
That way going forward we can see whether the next version might resolve our issues.

Thanks

Reply:
https://github.com/facebook/react-native/issues/18695
This kind of shows the issues. I think if we upgrade everything correctly we might be able to make it work. I think that the upgrades should be done in a separate branch though.